app/auth/views/oidc.py

```diff
@@ -60,19 +66,26 @@ def oidc_callback():
         flash("Please use another sign in method then", "warning")
         return redirect("/")
 
+    token_url = OIDC_TOKEN_URL
```

Comment:
As most IDP support the well-known URL, should we remove OIDC_TOKEN_URL, OIDC_USER_INFO_URL, etc and only keep OIDC_WELL_KNOWN_URL?

Reply:
I think it may be useful to leave that capability as a fallback, since there probably is something out there that does not expose a proper well-known information.

Reply:
in general, i agree with @viq that a fallback option could be useful for IDPs that don't expose it correctly - there are always some weird systems out there...
personally, i favor this one as it keeps flexibility for users. (and we'd also not break existing configurations.)

however, from the maintenance point of view it'd be a bit easier to maintain if we rely only on the well-known endpoint.

i believe this is a trade-off core maintainers need to decide.

Reply:
I think using well-known url is a great initiative as this avoids RP from having to hard code several values and even though if it isn't obligatory in the standard, it's supported by most IDP and we can decide to keep it as the only option supported in SL.